From: Stefan Raspl <raspl@linux.ibm.com>
Subject: net/smc: remove Kconfig warning
Patch-mainline: v4.14-rc1
Git-commit: d5b361b0dc3aa319522ef593b9bbf19ac3db23c5
References: FATE#325694, LTC#167874, bsc#1113480

Summary:     net/smc: SMC-R MVP
Description: Add latest upstream patches to push SMC-R to the MVP level

Upstream-Description:

             net/smc: remove Kconfig warning

             Now separate memory regions are created and registered for separate
             RMBs. The unsafe_global_rkey of the protection domain is no longer
             used. Thus the exposing memory warning can be removed.

             Signed-off-by: Ursula Braun <ubraun@linux.vnet.ibm.com>
             Signed-off-by: David S. Miller <davem@davemloft.net>

Signed-off-by: Stefan Raspl <raspl@linux.ibm.com>
Acked-by: Petr Tesarik <ptesarik@suse.com>
---
 net/smc/Kconfig |    4 ----
 1 file changed, 4 deletions(-)

--- a/net/smc/Kconfig
+++ b/net/smc/Kconfig
@@ -8,10 +8,6 @@ config SMC
 	  The Linux implementation of the SMC-R solution is designed as
 	  a separate socket family SMC.
 
-	  Warning: SMC will expose all memory for remote reads and writes
-	  once a connection is established.  Don't enable this option except
-	  for tightly controlled lab environment.
-
 	  Select this option if you want to run SMC socket applications
 
 config SMC_DIAG
